client/ayon_core/cli_commands.py

```diff
@@ -73,6 +74,11 @@ def publish(paths, targets=None, gui=False):
         import pyblish.api
         import pyblish.util
 
+        if not isinstance(path, str):
+            warnings.warn(
+                "Passing list of paths is deprecated.",
+                DeprecationWarning)
```

Comment:
It's deprecated, but with the logic changes will also just break completely. This should just be an error at this point right?

Reply:
This happens when you rush out and do only half of the thing you have in your mind. Thanks